content/gocourse-2019-05-melbourne.slide

```diff
@@ -1003,6 +1003,43 @@ Use `gofmt` or `goimport` as part of your editor/IDE setup
 - ...
 
 
+* JSON - (De-)Serialisation
+
+- `Encode` converts Go struct, pointer or primitive data type into JSON encoded stream
+- `Marshal` converts Go struct, pointer or primitive data type into JSON byte array
+- `Decode` and `Unmarshall` are the inverse functions to `Encode` and `Marshal`
+
+	func Marshal(v interface{}) ([]byte, error) 
+
+	func Unmarshal(data []byte, v interface{}) error 
+
+
+* JSON - Tags
```

Comment:
s/Tags/Field Tags/ - just for a bit more specificity. The `encoding/json` docs uses the term "field tag", although the go spec just uses "tag" (although it does call it "struct tag" in one place).